.gitignore

```diff
@@ -1,4 +1,7 @@
 .DS_Store
+.idea/
```

Comment:
I think we can leave out the gitignore file altogether. I will update the PR now.